lib/osf-components/addon/components/registries/review-metadata-renderer/styles.scss

```diff
@@ -0,0 +1,7 @@
+.ResponseValue {
+    composes: ResponseValue from '../schema-block-renderer/read-only/response/styles.scss';
+}
+
+.NoResponse {
+    composes: NoResponse from '../schema-block-renderer/read-only/response/styles.scss';
```

Comment:
Ditto ☝️re: double 'composes'. Also, do you you need to specify the `scss` extension?